config/lpdc-management/characteristics/form.ttl

```diff
@@ -198,6 +198,13 @@ ext:authorityPg a form:PropertyGroup;
       form:help "Meerdere overheden kunnen worden toegewezen";
       sh:order 30 ;
       sh:path lpdcExt:hasExecutingAuthority ;
+      form:validations [
+        a form:RequiredConstraint ;
+        form:language "en";
+        form:grouping form:Bag ;
+        sh:resultMessage "Dit veld is verplicht." ;
+        sh:path dct:description
```

Comment:
the sh:path specifies on what predicate the subject should be validated again. It might look redundant, given the way it is specified now, because inline, but it should actually `sh:path lpdcExt:hasExecutingAuthority ` too.
Actually, the fact that it didn't work, should be very obvious with even light testing: Even if this field would have been filled in, you wouldn't be able to send it